appeals/api/src/server/repositories/appeal.repository.js

```diff
@@ -136,6 +137,7 @@ const updateAppealById = (
 			...(caseStartedDate && { caseStartedDate }),
 			...(caseValidDate && { caseValidDate }),
 			...(applicationReference && { applicationReference }),
+			isGreenBelt,
```

Comment:
Just unsure about this... I thought this field is in the appellant case and lpaq, and no longer in the main Appeal table?